sw/device/exts/common/flash_link.ld

```diff
@@ -22,9 +22,12 @@ MEMORY {
 }
 
 /**
- * The stack starts at the end of RAM and grows down.
+ * The stack starts at the end of RAM and grows down. It is zeroed out so a size
+ * must be provided.
  */
+_stack_size = 0x2000; /* 8kb stack */
```

Comment:
This seems to be from a different approach?

Reply:
I needed a size to define a stack start and end point. Though could use an alternative approach to define stack end. For one thing this won't trigger any kind of error if the zero 8kb region runs into a section from the elf.

Reply:
It's needed for defining `_stack_end` for flash images, I think. (The one above is for the boot_rom).

Reply:
Ah I understand. Thanks!